python/afdko/agd.py

```diff
@@ -60,7 +60,7 @@ def parse(self, intext):
 	# Check a glyph for internal conflicts, problems, etc.
 	def check(self):
 		m = [] # holds reports of what was fixed
-		u = re.compile(r'uni([0-9A-F]{4})$') # Unicode value regexp
+		u = re.compile(r'u(ni)?([0-9A-F]{4,5})$') # Unicode value regexp
```

Comment:
You could use `?:` in `(?:ni)?` to make that group not count, which would avoid the group index adjustments further down.

Reply:
Cool! Didn’t know about that!